docs/DeveloperGuide.adoc

```diff
@@ -946,6 +946,15 @@ The following activity diagram illustrates what happens when a user executes `li
 .List Receipt Activity Diagram
 image::ExpenseSplitterListReceiptCommandActivityDiagram.png[]
 
+*Structure of List Receipt Command* +
+
+In this section, you will learn more about the relationships between objects related to the `listreceipt` command.
```

Comment:
I think should be “…objects related to the ListReceiptCommand…” rather than `listreceipt`